nixos/doc/manual/configuration/luks-file-systems.section.md

```diff
@@ -75,3 +75,10 @@ as [Trezor](https://trezor.io/).
 ```nix
 boot.initrd.luks.devices."/dev/sda2".fido2.passwordLess = true;
 ```
+
+With a passwordless setup described above, you can enable `askForPin` if your
```

Comment:
remember to run `./nixos/doc/manual/md-to-db.sh` after making changes to the manual

Reply:
Just did, thanks for the reminder!